### PR TITLE
News documentation US details & edit

### PR DESCRIPTION
**Pull request by samehaidi**

### CODE REVIEW

File: docs/GettingStarted.rst [hunk 1]
```diff
@@ -1205,6 +1205,22 @@ The article is shared into the space's activity stream.
 
 |image161|
 
+You can access the content of the article either by clicking on its title or by clicking on "read more".
+The details of the article are available on the current page and display all the information that has been entered as well as publication date and user.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
The details of the article are available on the current page and display all the information, including the publication date and the author.
```

File: docs/GettingStarted.rst [hunk 2]
```diff
@@ -1205,6 +1205,22 @@ The article is shared into the space's activity stream.
...
+
+|image162|
+
+You can change the content of the article using the edit icon. 
+The possible actions from the edit mode interface are "update", update and post" and "cancel". The buttons update and update and post are disabled until changes are made.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
The possible actions from the edit mode interface are `Update`, `Update and post` and `Cancel`. The buttons `Update` and `Update and post` are disabled until changes are made.
```

File: docs/GettingStarted.rst [hunk 3]
```diff
@@ -1205,6 +1205,22 @@ The article is shared into the space's activity stream.
...
+The update action allows you to make changes in the article without reposting it unlike the "update and post" action that allows to apply the changes in the article details and permits to rise up the article's preview in the activity stream.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
The `Update` action allows you to make changes in the article without reposting it unlike the `Update and post` action that allows to apply the changes in the article details and rise up the article's preview in the activity stream.
```

File: docs/GettingStarted.rst [hunk 4]
```diff
@@ -1205,6 +1205,22 @@ The article is shared into the space's activity stream.
...
+
+|image163|
+
+
+After updating the article, the information update user, update date and hour are available in the details as well as the different changes.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
When an article has been edited, the details view display the updated date and author besides initial information.
```

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]
@samehaidi the screenshot should be updated, they are not the latest version (update buttons on the left and big space between edit and share buttons).

**Comment by samehaidi** [comment]
@thomasdelhomenie  I just commited your suggestions. waiting for your remarks

**Review by thomasdelhomenie — APPROVED** [review]

**Pull request merged**